website/www/site/content/en/documentation/sdks/python-type-safety.md

```diff
@@ -210,7 +210,21 @@ However, if you enable runtime type checking, the code is guaranteed to fail at
 {{< code_sample "sdks/python/apache_beam/examples/snippets/snippets_test.py" type_hints_runtime_on >}}
 {{< /highlight >}}
 
-Note that because runtime type checks are done for each `PCollection` element, enabling this feature may incur a significant performance penalty. It is therefore recommended that runtime type checks are disabled for production pipelines.
+Note that because runtime type checks are done for each `PCollection` element, enabling this feature may incur a significant performance penalty. It is therefore recommended that runtime type checks are disabled for production pipelines. See the following section for a quicker, production-friendly alternative.
+
+### Faster Runtime Type Checking
+You can enable faster, sampling-based runtime type checking by setting the pipeline option `performance_runtime_type_check` to `True`.
+
+The is a Python 3 only feature that works by runtime type checking a small subset of values, called a sample, using optimized Cython code.
```

Comment:
We use `inspect.getfullargspec` in `typecheck.py L320` which is Py3 only. It could be refactored to support Py2, but like you say, it's not going to be around for much longer so it may not be worth the effort

Reply:
Ack. Sounds good to keep it as Python 3 only.